src/Components/Aspire.Npgsql/AssemblyInfo.LoggingCategories.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire;
+
+[assembly: LoggingCategories(
```

Comment:
We shouldn't need to split this out. Azure.Npgsql -> Npgsql, so the user will get the configuration schema for the underlying library as well.